tests/AutoReview/FixerFactoryTest.php

```diff
@@ -540,7 +540,7 @@ static function (array $priorityPair1, array $priorityPair2): int {
 
             static::assertSame($sortedDescription, $casesDescription);
         } else {
-            $this->addToAssertionCount(1);
+            $this->expectNotToPerformAssertions();
```

Comment:
The increase of assert count here was correct, the test was above it. The test fail message takes a long time to make so only in the case of failure this is done, if not a failure increasing the assert count is correct.